docs/references/nextjs/clerk-middleware.mdx

```diff
@@ -338,38 +338,45 @@ The `clerkMiddleware()` function accepts an optional object. The following optio
 
   ---
 
+  - `organizationSyncOptions?`
+  - <code>[OrganizationSyncOptions](#organization-sync-options) | undefined</code>
+
+  Used to activate a specific [organization](docs/organizations/overview) or [personal account](https://clerk.com/glossary#personal-account) based on URL path parameters. If there's a mismatch between the active organization in the session (e.g., as reported by [`auth()`](/docs/references/nextjs/auth)) and the organization indicated by the URL, the middleware will attempt to activate the organization specified in the URL.
+
+  ---
+
   - `proxyUrl?`
   - `string`
 
   Specify the URL of the proxy, if using a proxy.
 
   ---
 
-  - `signInUrl?`
+  - `publishableKey`
   - `string`
 
-  An alternative sign in URL.
+  The Clerk publishable key for your instance. This can be found in your Clerk Dashboard on the [**API Keys**](https://dashboard.clerk.com/last-active?path=api-keys) page.
 
   ---
 
-  - `signUpUrl?`
+  - `secretKey?`
   - `string`
 
-  An alternative sign up URL.
+  The Clerk secret key for your instance. This can be found in your Clerk Dashboard on the **[API Keys](https://dashboard.clerk.com/last-active?path=api-keys)** page. The `CLERK_ENCRYPTION_KEY` environment variable must be set when providing `secretKey` as an option. For more information, refer to [Dynamic keys](#dynamic-keys) section.
```

Comment:
```suggestion
  The Clerk secret key for your instance. This can be found in your Clerk Dashboard on the **[API Keys](https://dashboard.clerk.com/last-active?path=api-keys)** page. The `CLERK_ENCRYPTION_KEY` environment variable must be set when providing `secretKey` as an option. For more information, refer to the [Dynamic keys](#dynamic-keys) section.
```